src/robot/model/body.py

```diff
@@ -199,7 +215,7 @@ def _filter(self, types, predicate):
             items = [item for item in items if predicate(item)]
         return items
 
-    def flatten(self):
+    def flatten(self) -> 'list[BodyItem|Try|If]':
```

Comment:
This method effectively removes all `Try` and `If` objects from the returned list so they shouldn't be listed in the return type either.